lib/import/admin.js

```diff
@@ -128,29 +115,29 @@ function save_topojson(country_code, feature_collection, geojson, verbose) {
  * Iterate through features, assign values to attributes, and save. NOTE: this
  * mutates `feature_collection`.
```

Comment:
Looks like this comment about mutating feature_collection is out-of-date - could you remove that?